weather-app/src/app/fovorites-list/fovorites-list.component.ts

```diff
@@ -1,13 +1,13 @@
 import { FavoriteCityListService } from './../services/favorite-city-list.service';
```

Comment:
You have a typo in the file name: [fOvorites-list.component.ts]

Reply:
and the folder name, actually )

Reply:
It was merged earlier, but fixed)